### ISSUE TITLE
Make onrr.gov issues based off the rerun OCIO report

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2689]
Make issues at the violation level, include the notes, number of instnces, priority & severity. Send Jeff a copy of the excel export of violations
